core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -1100,10 +1100,8 @@ class ReplicaManager(val config: KafkaConfig,
             leaderLogEndOffset = readInfo.logEndOffset,
             followerLogStartOffset = followerLogStartOffset,
             fetchTimeMs = fetchTimeMs,
-            readSize = adjustedMaxBytes,
```

Comment:
Do we still need to compute `adjustedMaxBytes`? Also, do you know why we don't need `readSize` anymore? What change made it unnecessary?

Reply:
We can get the read size already from the `Records` object. The code must have been changed at some point to use this. It looks like `adjustedMaxBytes` is still sent through in the call to `Partition.readRecords`.

Reply:
Thanks, makes sense.